src/app/ui.cljs

```diff
@@ -38,6 +38,10 @@
   (reset-game!))
 
 
+(defn use-hint! []
+  (r/rswap! app-state assoc :game/game (game/hint (:game/game @app-state))))
```

Comment:
This is valid and works but the same can also be written more concisely using `update` instead of `assoc`:

```clojure
(r/rswap! app-state update :game/game game/hint)
```

`update` applies the given function (`game/hint`) to whatever is found under the key (`:game/game`) and the result will be updated as the new value for that key.

Reply:
That's a lot simpler. Thanks! I've updated the code.